.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run lint
+composer cs -- --runtime-set ignore_warnings_on_exit 1
```

Comment:
Why ignore warnings? Letting them go through is then just noise for the next person.

I'd also like to see us add `--severity=1` (and fix up the issues that appear as well).

Reply:
My rationale was: if the were not severe enough to be deemed an error, why should the be severe enough to block commits?

In the same way that a PHP warning alerts (if you're configured to see them) but continues executing.

> I'd also like to see us add --severity=1 (and fix up the issues that appear as well).

Does that make what are now phpcs warnings into errors?

Reply:
I stick with Gary on this one -- unless there's a very good reason (and in that case, you should be using a per-line ignore), I would treat warnings as errors. This would result in a more cohesive codebase. This is in line with `--severity=1`.

Reply:
works for me